server/app/lib/actions/fusor/content/manage_content.rb

```diff
@@ -105,6 +96,26 @@ def retrieve_deployment_repositories(organization, product_content)
           repos
         end
 
+        def configure_activation_keys(deployment, hostgroup)
+          content = SETTINGS[:fusor][:content]
+
+          hostgroup[:host_groups].each do |subgroup|
+            next unless subgroup[:activation_key]
+
+            content_key = subgroup[:activation_key][:content]
+            product_content = content[content_key]
+
+            # find the subset of repos that belong to this activation key
+            repositories = retrieve_deployment_repositories(deployment.organization, product_content)
```

Comment:
Previously, we filtered for 'yum' repositories, do we want to do the same?  (This would omit docker, puppet, file..etc. repositories, which aren't typically used by an activation key)